go/vt/tabletserver/messager_cache.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2017, Google Inc. All rights reserved.
```

Comment:
First 2017 copyright! ;)

Seriously, the Messager stuff is a few files, how about moving them to a tabletserver/messager subpackage?

Also, Messager is not an english word, Messenger instead? Or MessageManager?

(Note I like it, it's the french word for it!)